src/style.css

```diff
@@ -99,6 +99,18 @@ main {
   align-self: flex-end;
 }
 
+.message {
+  font-size: 1.6rem;
+}
+
+.success {
+  color: green;
+}
+
+.error {
+  color: tomato;
+}
+
```

Comment:
- [ ] [OPTIONAL] I think there are a couple of spelling errors on your page. Kindly fix them. I believe it is spelled as `Recent` and the one on the right should be `Add your score`. 😎️

![image](https://user-images.githubusercontent.com/17243325/215949531-e09ea995-8ff2-4baa-8c0d-636209f56b49.png)

Reply:
Thank you. I will implement the recommended change.